docs/static/releasenotes.asciidoc

```diff
@@ -36,6 +37,46 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-10-1]]
+=== Logstash 7.10.1 Release Notes
+
+==== Notable issues fixed
+
+===== Fix docker image metadata
+Removed non-OCI compliant freeform metadata labels, started using consistent build date for all the docker images and dockerfiles. https://github.com/elastic/logstash/pull/12451[#12451]
+Reintroduces the labels for the ubi8 image only. https://github.com/elastic/logstash/pull/12500[#12500]
+
+===== Exclude file with non-commercial license
+Bundled did_you_mean gem had a CC NonCommercial license in an included file, these files has been removed. https://github.com/elastic/logstash/pull/12460[#12460]
+
+===== Support recreation of same pipeline through centralized pipeline management
+When users attempted to delete and recreate a pipeline with the same identifier and configuration, Logstash was unable to pick up the new pipeline. https://github.com/elastic/logstash/issues/12387[#12387]
```

Comment:
Regarding what's notable about this release I think we focus on just the agent reloading issue:

```suggestion
===== Support recreation of same pipeline through centralized pipeline management
When users attempted to delete and recreate a pipeline with the same identifier and configuration, Logstash was unable to pick up the new pipeline. https://github.com/elastic/logstash/issues/12387[#12387]
```